### ISSUE TITLE
Add a contributing.md?

### ISSUE THREAD

**Issue by abhsag24** [sdispater/orator#292]
i absolutely love the project idea and implementation. Would love to contribute. But cant find an irc? or contribution standards? any pointers? also maybe a good first issue label might be a great idea?

**Comment by psincraian** [comment]
I want to contribute too.

I have start by setting up the environment and is not so easy because you need the databases. To make things easier for everyone I will try to user docker and document all the process.

**Comment by zardilior** [comment]
Is this still on? Because I am also interested on contributing and we should definetely go for a contributing.md as well as instructions on mounting the dev environment.

**Comment by josephmancuso** [comment]
@zardilior I got pretty much full access to this repo from @sdispater but he has been really busy with his other projects like Poetry and Pendulum.

I am also very busy with @masoniteframework https://github.com/MasoniteFramework/masonite and don't really have the time to learn and work the codebase.

If Orator has a dedicated maintainer it would go through the roof. If you are interested in taking on that challenge you can join the Masonite slack group and you can bounce questions and work with us

**Comment by zardilior** [comment]
@josephmancuso I would need a second guy to become a dedicated mantainer myself. So how about going into the Masonite slack group?

**Comment by zardilior** [comment]
@josephmancuso If you give me access I will, gather a community around orator, fix all easy to fix bugs, learn the codebase and make docs to explain it, pass most issues to stack overflow to popularize the framework :D So how about that? Do I win the mantainer election?

**Comment by josephmancuso** [comment]
@zardilior yeah that'd be great. I don't have the rights to give people access but I'll message @sdispater to give you access

**Comment by zardilior** [comment]
@josephmancuso thanks man!! Also how do I enter the slack you were mentioning?

**Comment by josephmancuso** [comment]
http://slack.masoniteproject.com

**Comment by zardilior** [comment]
@abhsag24 @psincraian you are still interested? Let's fork the project and make a slack!! join the masonite too!

**Comment by abhsag24** [comment]
@zardilior yep. I'm also working on an annotate library which adds model description comments on each model file kinda like https://github.com/ctran/annotate_models . I'd be interested in taking this forward as well!

**Comment by josephmancuso** [comment]
@abhsag24 this might help https://github.com/MasoniteFramework/core/blob/2.2/masonite/commands/ModelDocstringCommand.py

**Comment by abhsag24** [comment]
nice! can we use it independently outside of masonite?

**Comment by josephmancuso** [comment]
umm not exactly. You could use Cleo and register it with the Cleo command class manually and it should work but might want to take it and make it more adaptable to your application. Unless that works for you
